.expeditor/scripts/shared.sh

```diff
@@ -339,3 +339,68 @@ push_current_branch() {
 }
 
 
+macos_install_bootstrap_package() {
+    echo "--- Installing mac bootstrap package"
+    # subscribe to releases: https://github.com/habitat-sh/release-engineering/issues/84
+    bootstrap_package_version="$(cat MAC_BOOTSTRAPPER_VERSION)"
+    bootstrap_package_name="mac-bootstrapper-${bootstrap_package_version}-1"
+    curl "https://packages.chef.io/files/stable/mac-bootstrapper/${bootstrap_package_version}/mac_os_x/10.13/${bootstrap_package_name}.dmg" -O
+    sudo hdiutil attach "${bootstrap_package_name}.dmg"
+    sudo installer -verbose -pkg "/Volumes/Habitat macOS Bootstrapper/${bootstrap_package_name}.pkg" -target /
+    sudo hdiutil detach "/Volumes/Habitat macOS Bootstrapper"
+    brew install wget
+    export PATH=/opt/mac-bootstrapper/embedded/bin:/usr/local/bin:$PATH
+    
+}
+
+macos_use_cert_file_from_linux_cacerts_package() {
+    # BEGIN CERTIFICATE SHENANIGANS
```

Comment:
With all the SSL changes in recent months, I wonder if all this is still necessary thinking

(This doesn't need to be addressed for this PR, though.)